jdi-light-vuetify/src/main/java/com/epam/jdi/light/vuetify/elements/common/Switch.java

```diff
@@ -61,6 +68,109 @@ public boolean isEnabled() {
         return !hasClass("v-input--is-disabled");
     }
 
+    @JDIAction("Get {name}'s color")
+    public String hasColor() {
```

Comment:
Please use naming without "has" if method returns not boolean value

Reply:
done